content/en/providers/check-in/sp/_index.md

```diff
@@ -482,14 +480,17 @@ usual but includes the `code_verifier` secret generated in the first request.
 Example request:
 
 ```shell
-curl -X POST "https://aai.egi.eu/oidc/token" \
+curl -X POST "${TOKEN_ENDPOINT}" \
 -d "grant_type=authorization_code" \
```

Comment:
```suggestion
  -d "grant_type=authorization_code" \
```

Reply:
Resolved by commit [Add indent to curl data parameter](https://github.com/EGI-Federation/documentation/pull/332/commits/d4c45e1af19edd9a4d4cdaa64e897a218160bc87)